source/MaterialXCore/Element.cpp

```diff
@@ -81,6 +81,69 @@ bool Element::operator!=(const Element& rhs) const
     return !(*this == rhs);
 }
 
+bool Element::isEquivalent(ConstElementPtr rhs, ElementEquivalenceOptions& options) const
+{
+    if (getCategory() != rhs->getCategory() ||
```

Comment:
I'm curious why category and name are specialized here - won't they also be covered by the attribute comparison below?

Reply:
No, they are stored in different class member data.